src/components/RecipeDetails.tsx

```diff
@@ -4,58 +4,163 @@ import {
   IInstructionsIngredients,
   ISteps,
   IEquipment,
+  IRecipe,
 } from "../interfaces/Recipe";
-import { v4 as uuidv4 } from 'uuid';
+import { v4 as uuidv4 } from "uuid";
+import styled from "styled-components";
+import PlusIcon from "../assets/plus.svg";
 
 interface RecipeDetailsProps {
   recipeInstructions: IInstructions[];
-  selectedRecipeImage: string;
+  selectedRecipe: IRecipe[];
 }
 
+interface ImageProps {
+  image: string;
+}
+
+const FoodImage = styled.div<ImageProps>`
+width: 500px;
+background-color: #fff;
+background-image:  ${props =>`url(${props.image})`};
+background-size: cover;
+background-repeat: no-repeat;
+background-position: center;
+border-radius: 50%;
+height: 500px;
+position: absolute;
+right: 0;
+transform: translateX(15%) translateY(-15%);
+`;
+
+const RecipeDetailsCard = styled.div`
+  background-color: #ef8080;
+  border-radius: 15px;
+  position: relative;
+  padding-left: 70px;
+  overflow: hidden;
+`;
+
+const AddBtn = styled.div`
+  width: 50px;
+  height: 50px;
+  background-color: #fff;
+  position: absolute;
+  top: 0;
+  left: 0;
+  border-radius: 15px 0 15px 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+const RecipeInfo = styled.div`
+  width: 550px;
+  height: 500px;
+  margin-top: 100px;
+  text-align: start;
+  color: #fff;
+
+  h2 {
```

Comment:
Just be careful if you were to make these into style-components for the typography. Don't know if this would pick it up

Reply:
Will keep that in mind. Thanks for the tip!